src/networks.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
   LOCAL: 'local',
-  EVONET: 'evonet',
+  EVONET: 'evonet-6',
```

Comment:
It should stay `evonet` otherwise other things won't work.

Reply:
OK, moved the version to the config schema